threema-connector-test/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>threema.connector.test</groupId>
+  <artifactId>threema-connector-test</artifactId>
+  <version>10.0.0-SNAPSHOT</version>
+  <packaging>iar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>threema.connector</groupId>
+      <artifactId>threema-connector</artifactId>
+      <version>${project.version}</version>
+      <type>iar</type>
+    </dependency>
+    <dependency>
+      <groupId>com.axonivy.ivy.test</groupId>
+      <artifactId>unit-tester</artifactId>
+      <version>11.1.0</version>
```

Comment:
you should downgrade this to the current lts 10 version.

Reply:
downgraded to 10.0.0 as per https://mvnrepository.com/artifact/com.axonivy.ivy.test/unit-tester